### ISSUE TITLE
why set TEST.USE_GT_BBOX False

### ISSUE THREAD

**Issue by SamuelSJTU** [leoxiaobin/deep-high-resolution-net.pytorch#20]
you provide coco test command
```
python tools/test.py \
    --cfg experiments/coco/hrnet/w32_256x192_adam_lr1e-3.yaml \
    TEST.MODEL_FILE models/pytorch/pose_coco/pose_hrnet_w32_256x192.pth \
    TEST.USE_GT_BBOX False
```
But I found I can get high accuracy only if  I set it true.  Otherwise, the accuracy is almost zero.

**Comment by mrbulb** [comment]
I test your command(set TEST.USE_GT_BBOX False), and achieve the AP of 0.744, which is the same in the `README.md`. So I think there may be someting wrong with your code.

For why setting TEST.USE_GT_BBOX False, in my view, HRNet adopts the top-down method for pose estimation.  First use a detector generate the person box, and then the HR-Net is used to predict the person keypoints. 

So person detection result is a keypoint that determines the final keypoints performance. If your set TEST.USE_GT_BBOX True, it means you use ground truth as the input of the second stage. And it's nonsense for the top-down method's evaluation.

**Comment by zhj123169** [comment]
I have the same question.Have you saved it? @SamuelSJTU

**Comment by zhj123169** [comment]
I have the same question.Have you solved it?? @SamuelSJTU

**Comment by SamuelSJTU** [comment]
@mrbulb I read the code. HRNet use bbox to crop the image in training and validation stage. If TEST.USE_GT_BBOX set to false, it uses bbox annotation in COCO_val2017_detections_AP_H_56_person.json. Otherwise, it uses bbox annotation in coco_val.json. This two annotation data is different. I think it dosen't provide annotation for human pose. Maybe I get wrong data for  COCO_val2017_detections_AP_H_56_person.json.
